english/swiper/dora.md

```diff
@@ -0,0 +1,4 @@
+You see two tunnels . Before them is Dora asking you the way to find the marshmellow palace .She asks you which tunnel to take that has no swiper waiting to swipe the marshmellows.oh man!
```

Comment:
It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.